lib/ansible/modules/windows/win_environment.ps1

```diff
@@ -43,44 +32,38 @@ $result = @{
     value = $value
```

Comment:
It's a pity this module sets `name`, `level` and `value` in the return arguments now we want to move away from this. Should we remove them considering it is preview/community right now and set the example right?

Reply:
Yes, that's my objection as well. Same for win_uri (and most of the older modules).

Personally I don't mind removing it because I cannot believe anyone using it.

Reply:
I can add a note that these are there for backward compatibility.

Reply:
In this case I would remove it. I feel funny about others like win_get_url as they may contain info that people use. This is just the invocation args.

Reply:
We could retain $value, since it's useful to see the old and new value in the same output.

Reply:
I'll leave it up to you, I personally don't think it is too useful but can sort of see it.